### ISSUE TITLE
Extended User Fields out of order?

### ISSUE THREAD

**Issue by modxbot** [modxbot/migrate#2582]
enigmatic_user created Redmine issue ID 2582

Whatever I do in the Extended User Fields tab, it doesn't have any consequences. When trying to add a container or an attribute, the respective dialogues just stay open when I click on the "Save" buttons; I have to cancel the operations to close the dialogues.

Did I just misunderstand something (see Bob Ray's suggestions regarding the usability: http://bugs.modx.com/issues/263 and http://bugs.modx.com/issues/157) or are these dialogues just out of order?

**Comment by keiron77** [comment]
keiron77 submitted:

I have also recently experienced this. I saw it working on a previous version, but since upgrading extended fields fail to work in the same way.

LAMP server
Firefox and Chrome
OS X

This a quite a critical bug for me as I was using the extended fields in the current project.
Recovering JSON data from the database isn't a problem, but creating/saving/editing in the manager is broken. I suspect it's a JS problem in the manager.
